content/blog/2021/04/2021-04-07-contributhon-participants.adoc

```diff
@@ -0,0 +1,78 @@
+---
+:layout: post
+:title: "SheCodeAfrica Contributhon participants"
+:tags:
+- events
+- community
+:author: markewaite
+:sig: docs
+opengraph:
+  image: /images/post-images/2020-04-contributhon-participants/shecodeafrica-participants-opengraph.png
+---
+
+The link:https://sites.google.com/shecodeafrica.org/contributhon[SheCodeAfrica Contributhon] started April 1, 2021 with participants from Nigeria, Kenya, and Cameroon.
+Twice weekly mentoring sessions are listed in the link:/events[events calendar].
+We've already received the first pull request to improve the Pipeline documentation of key plugins.
+We're looking forward to more pull requests and more improvements throughout April, 2021.
+
+We'd like to introduce our participants so that you recognize them and can welcome them during code review and online chat.
+
+== Onyinye Ezike
+
+image:/images/post-images/2020-04-contributhon-participants/onyinye-ezike.jpg[Onyinye Ezike, width=400]
+
+Onyinye is based in Lagos, Nigeria.
+She's a junior fullstack software developer who is very passionate about learning.
+She uses Angular, React, Nodejs, and Spring Boot.
+She has spent the last two years building up herself in software development and she's hoping to become a world-class software developer.
+
+You'll recognize her contributions as link:https://github.com/Onyimatics[`Onyimatics`] on GitHub and on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20onyimatics[Jenkins issue tracker].
+
+== Sharon Jebitok
+
+image:/images/post-images/2020-04-contributhon-participants/sharon-jebitok.jpg[Sharon Jebitok, width=400]
+
+Sharon is based in Kenya.
+She's a software development student at Microverse, a remote school for software developers.
+She has been an active member of She Code Africa and she's presently a mentee on She Code Africa's Contributhon Program with the Jenkins project.
+She has spent one year in the software industry since she switched her career into tech, learning to write meaningful code and collaborate with other developers/designers and the tech community.
+
+You'll recognize her contributions as link:https://github.com/jebitok-dev[`jebitok-dev`] on GitHub and as link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20jebitok16_[`jebitok16_`] on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20jebitok16_[Jenkins issue tracker].
+
+== Esther Ejidike
+
+image:/images/post-images/2020-04-contributhon-participants/esther-ejidike.jpg[Esther Ejidike, width=400]
+
+Esther is a frontend developer based in Lagos, Nigeria.
+She is one of the participants in the She Code Africa Contributhon Open Source Program working with Jenkins.
```

Comment:
```suggestion
```

Same as everyone else in this list :) Some extra info would be nice, but not a blocker.

Reply:
Agreed.  I would really like a bio from Esther and would like to assure that I have the correct GitHub ID and Jenkins ID before we publish.

Reply:
Hey Mark,

I have shared with you my bio. I am not sure if you have seen it.  You can find it here: https://docs.google.com/document/d/1COk83GZSYgnOkCOU5qNsbJvedE6nJmtVxrKQukw-MtY/edit#

Thanks